src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -295,4 +295,53 @@ public void saveEventInformation(List<Event> events, PreparedQuery results) {
   }
 
 
+  /**This function returns one Event given a query**/
```

Comment:
The description of the function seems incorrect, I would change this as an event is not being returned in saveIndividualEvent.